CONTRIBUTING.md

```diff
@@ -56,6 +56,7 @@ Before submitting a pull request, please ensure:
 
 * Code has been formatted using the provided .clang-format file.
 * Naming conventions are following the recommendations.
+* The project has been built in developer mode (`DEVELOPER_MODE` is set to `ON`).
```

Comment:
Should I also expand on helping to get potential contributors setup with presets similar to https://github.com/friendlyanon/cmake-init-shared-static/blob/master/HACKING.md ? Having an almost copy-paste user preset could help a ton, plus the informative note about VS being greedy about configuring with whatever preset if finds first.

Reply:
Yes, please.  Thanks!